qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_mirror_to_rbd:
+    only Linux
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_to_rbd
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data1 = 2G
+    image_size_mirror1 = ${image_size_data1}
+    image_format_mirror1 = raw
+    image_name_data1 = data1
+    image_name_mirror1 = mirror1
+    storage_pool_mirror1 = rbd
+    rbd_pool_name_mirror1 = ${storage_pool_mirror1}
```

Comment:
remove this line for rbd_pool_name will be imported from outside

Reply:
Have updated, thanks zhenchao